src/app.test.js

```diff
@@ -1,14 +1,35 @@
-import { exitBrowser } from './lib/htmlToPdf';
+import { exitBrowser, initBrowser } from './lib/htmlToPdf';
+import { fromBuffer } from 'pdf2pic';
+import { OPG_TEST_LETTER } from './assets/opgTestLetter';
+import pixelmatch from 'pixelmatch';
+import { PNG } from 'pngjs';
+import { readFileSync, writeFileSync } from 'node:fs';
 
 import request from 'supertest';
 import app from './app';
 
 const testHtml = `<html><head></head><body><p><a href="/home" class="govuk-link">Test with no links</a></p></body></html>0i9`;
 
+beforeEach(async () => {
+  await initBrowser();
+});
+
 afterAll(async () => {
   await exitBrowser();
 });
 
+describe('Given the app gets an api request to an endpoint', () => {
+  describe('GET /health-check', () => {
+    test('It should respond with a http status code of 200', async () => {
+      await request(app.handler)
+        .get('/health-check')
+        .timeout(8)
```

Comment:
When I ran this locally, it timed out until I increased this limit. Not necessarily a big deal if it works fine in CI, but might be worth being a bit more lenient to avoid false negatives.

Reply:
Yep, we can always readjust if it gets flaky in CI. I've approved the PR ✅